Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/TranslationHandles.cs

```diff
@@ -46,11 +46,11 @@ protected override Quaternion GetRotationRealignment(int handleIndex)
             // Align handle with its edge assuming that the prefab is initially aligned with the up direction 
             if (VisualUtils.FaceAxisType[handleIndex] == CardinalAxisType.X)
             {
-                return Quaternion.FromToRotation(Vector3.up, directionSign * Vector3.right);
+                return Quaternion.FromToRotation(Vector3.forward, directionSign * Vector3.right);
```

Comment:
Was the handle rotation issue a regression or has this rotation always been in the wrong direction?

Reply:
Looks like the rotation issue is not a regression based on this comment: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/9346#issuecomment-782428530